docs/install/sandboxes.rst

```diff
@@ -297,10 +297,12 @@ This image has all software needed to build envoy. From your envoy directory::
 
   $ pwd
   src/envoy
-  $ docker run -t -i -v <SOURCE_DIR>:/source lyft/envoy-build:latest /bin/bash -c "cd /source && ci/do_ci.sh normal"
+  $ ./ci/run_envoy_docker.sh './ci/do_ci.sh bazel.dev'
```

Comment:
I will update the comment below to mention that it runs tests as well. And make the link to the readme to find other do_ci targets.